tycho-its/projects/tycho937/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>tycho937</groupId>
+	<artifactId>tycho937</artifactId>
+	<packaging>eclipse-plugin</packaging>
+	<version>1.0.0-SNAPSHOT</version>
+
+	<properties>
+		<tycho-qualifier>yyyyMMdd-HHmm</tycho-qualifier>
+	</properties>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-packaging-plugin</artifactId>
+				<version>${tycho-version}</version>
+				<configuration>
```

Comment:
I think you need to add the Tycho plugin here not only the packing plugin!

Reply:
yes now test is green, so it does not show the problem

Reply:
is it possible jenkins and local(mac) behave differently?